src/MicrosoftTeams.ts

```diff
@@ -282,6 +284,42 @@ namespace microsoftTeams {
         }
     }
 
+    /**
+     * Registers a handler for user presses of the Team client's back button. Experiences that maintain an internal
+     * navigation stack should use this handler to navigate the user back within their frame. If an app finds
+     * that after running its back button handler it cannot handle the event it should call the navigateBack
+     * method to ask the Teams client to handle it instead.
+     * @param handler The handler to invoke when the user presses their Team client's back button.
+     */
+    export function registerBackButtonHandler(handler: () => void): void {
+        ensureInitialized();
+
+        backButtonPressHandler = handler;
+    }
+
+    function handleBackButtonPress(): void {
```

Comment:
it would be nice if ** backButtonPressHandler** could signal whether to perform "navigateBack" or suppress it. so the client of the SDK does not have to go ahead and call navigateBack();

similar to when you have a click event and you say: event.preventDefault() [maybe returning false works too];

Reply:
Done; if the tab returns false from its back button handler we will automatically call navigateBack(); if it returns true then we assume the back button press was handled and we shouldn't do anything.